snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -355,19 +358,45 @@
         }
     }
 
-    // Download the user's country using the geolocation service.
-    function downloadUserCountry() {
-        $script(GEO_URL, 'geo');
-        $script.ready('geo', function() {
-            try {
-                gSnippetsMap.set('geoCountry', geoip_country_code());
-                gSnippetsMap.set('geoLastUpdated', new Date());
-            } catch (e) {
-                // Most likely failed to load JS file. Continue on without us,
-                // we'll try again next time.
-            }
-        });
-    }
+    {% if client.channel == 'release' %}
```

Comment:
Discussed in IRC but answering also here for the record: 

This is to limit the impact of this change to non-release users and not to a specific Fx version. It will give us the opportunity to test and verify on beta/aurora/nightly and make sure that MLS can handle the load.